packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -2113,6 +2113,7 @@
   "LDAP_Sync_User_Data_GroupsMap": "User Data Group Map",
   "LDAP_Sync_User_Data_GroupsMap_Description": "Map LDAP groups to Rocket.Chat user roles <br/>As an example, `{\"rocket-admin\":\"admin\", \"tech-support\":\"support\"}` will map the rocket-admin LDAP group to Rocket's \"admin\" role.",
   "LDAP_Test_Connection": "Test Connection",
+  "LDAP_Connection_success":"Connection Successfull",
```

Comment:
The correct i18n key is actually `Connection_success`, as you can see being user at:
https://github.com/RocketChat/Rocket.Chat/blob/7d6b9a86c3f920b331373d7d84fa674b17ab9525/app/ldap/server/testConnection.js#L38

Reply:
So in the i18n key I just need to put Connection_sucess , Am i correct ?